client/src/hooks/useParks.js

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from "react";
+import {
+  getItemsFromLocalStorage,
+  saveJSONToLocalStorage,
+} from "../utils/itemStorage";
+import parksData from "../parks.json";
+
+function initialLoad() {
+  const retrieveArray = getItemsFromLocalStorage("parkData");
+  if (retrieveArray.length === 0) {
+    return parksData;
+  } else {
+    return retrieveArray;
+  }
+}
+
+export default function useParks() {
+  const [parkData, setParkData] = useState(initialLoad());
```

Comment:
React's `useState` will always call a function if it passed as initial value
```suggestion
  const [parkData, setParkData] = useState(initialLoad);
```